tests/queries/0_stateless/03261_minmax_indices_by_default.sql

```diff
@@ -11,11 +11,14 @@ CREATE TABLE tbl1
     key Int,
     x Int,
     y Int,
-    INDEX x_idx x TYPE minmax
+    INDEX x_idx x TYPE minmax GRANULARITY 1
```

Comment:
Am I right that this is not required to fix flakiness? (It is OK to keep it, just curiosity)

Reply:
Yes, that's not required, I've set it "just in case"